libweb3jsonrpc/Eth.cpp

```diff
@@ -428,7 +428,22 @@ string Eth::eth_sendRawTransaction( std::string const& _rlp ) {
     // will be checked as a part of transaction import
     Transaction t( jsToBytes( _rlp, OnFailed::Throw ), CheckTransaction::None, false,
         EIP1559TransactionsPatch::isEnabledInWorkingBlock() );
-    return toJS( client()->importTransaction( t ) );
+    try {
+        return toJS( client()->importTransaction( t, TransactionBroadcast::BroadcastToAll ) );
+    } catch ( PendingTransactionAlreadyExists& ) {
+        throw std::runtime_error( "Transaction with same nonce already exists in the queue." );
+    } catch ( TransactionAlreadyInChain& ) {
+        // make it similar to what geth does
+        throw std::runtime_error( "Nonce too low." );
+    } catch ( InvalidNonce& ) {
+        if ( !client()->chainParams().sChain.multiTransactionMode ) {
+            // make it similar  to what geth does
+            throw std::runtime_error( "Nonce in the future." );
+        } else {
+            // make it similar  to what geth does
+            throw std::runtime_error( "Invalid nonce." );
+        }
+    }
```

Comment:
Error handling for eth_sendRawTransaction is already implemented here:
https://github.com/skalenetwork/skaled/blob/develop/libweb3jsonrpc/rapidjson_handlers.cpp#L55

Reply:
Fixed

Reply:
Fixed